packages/teleterm/src/ui/components/ServerSideSearchPanel/ServerSideSearchPanel.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Text, Box, Flex, Link } from 'design';
+import ClusterSearch from 'teleterm/ui/DocumentCluster/ClusterResources/ClusterSearch';
+import { StyledPanel } from 'design/DataTable';
+import Toggle from 'teleport/components/Toggle';
+import Tooltip from '../Tooltip';
+
+const GUIDE_URL =
+  'https://goteleport.com/docs/setup/reference/predicate-language/#resource-filtering';
+
+export function ServerSideSearchPanel({
```

Comment:
Would it be possible to avoid copying this component?
It seems to be pretty similar to the one from the WebUI. Probably the only thing we would have to replace is the logic in `useServersideSearchPanel`.